docs/architecture-docs.md

```diff
@@ -0,0 +1,29 @@
+# Ground Android app architecture
+
+The Ground Android app is written in Java using the [Android Jetpack](https://developer.android.com/jetpack) and [RxJava](https://github.com/ReactiveX/RxJava) libraries. This page provides an overview of the high-level app architecture, including key components and libraries.
+
+<!-- Editable image source: https://docs.google.com/drawings/d/1UjetRJsudLHg3YsWaf0sKqHfmrPNNBH4UKI7Ivajx-0/ -->
+![Ground Android app architecture diagram](android-architecture-diagram.png)
+
+## Components
+
+The app is comprised of the following components:
+
+- **MainActivity**: Ground is a single-activity app. The main [activity](https://developer.android.com/guide/components/activities/intro-activities) contains all other app components.
+- **Fragment**: Each part of the app interface is defined as a [fragment](https://developer.android.com/guide/components/fragments). Examples of fragments are the home screen and the map interface.
+- **ViewModel**: Each fragment has a corresponding [ViewModel](https://developer.android.com/topic/libraries/architecture/viewmodel), which encapsulates its UI state, logic, and functionality, as well as the lifecycle callback methods.
+- **Managers**: The app managers abstract access to system services directly for the ViewModels. Examples of services handled by managers are location services and camera access.
+- **Models**: Entity representations used by ViewModels, such as the app's representations of map concepts, are defined as models. Models allow the app to remain platform agnostic because we can update model definitions without affecting ViewModels or other app components. Examples of entities defined models are the app's representation of offline imagery.
```

Comment:
Models -> Model (singular, since there's only one "model")

> Models allow the app to remain platform agnostic because we can update model definitions without affecting ViewModels or other app components.

I'd replace this with something like "the .model package represents data across the various components of app",.

More here:
https://developer.android.com/jetpack/guide#drive-ui-from-model

Reply:
Done